packages/ui/src/types/authenticator/stateMachine/event.ts

```diff
@@ -19,6 +19,7 @@ export type AuthEventTypes =
   | 'AUTO_SIGN_IN'
   | 'AUTO_SIGN_IN_FAILURE'
   | 'SIGN_IN'
+  | 'SIGNED_IN'
```

Comment:
Is this new event type needed, couldn't it be covered by `done.invoke.signInActor`? Also I'm not seeing `SIGNED_IN` used in this PR but may have missed it.

Reply:
Ah yes this was just for testing, will remove